optimum/onnxruntime/modeling_ort.py

```diff
@@ -283,6 +283,22 @@ def _from_transformers(
         # 3. load normal model
         return cls._from_pretrained(save_dir.as_posix(), **kwargs)
 
+    def prepare_onnx_inputs(self, **kwargs):
```

Comment:
@philschmid What about something like that? This one took `{'Inference_runtime': 16.687}`, closer to what we had compared to my previous try.